saleor/dashboard/order/views.py

```diff
@@ -591,7 +591,7 @@ def fulfill_order_lines(request, order_pk):
         request.POST or None, queryset=FulfillmentLine.objects.none(),
         initial=initial)
     all_forms_valid = all([line_form.is_valid() for line_form in formset])
-    if all_forms_valid and form.is_valid():
+    if all_forms_valid and formset.is_valid() and form.is_valid():
```

Comment:
Maybe we should rename `all_forms_valid` to sth like `are_line_forms_valid` - now this naming suggests that literally all forms used here are valid while in the `if` check we still check `formset` and `form`.

Reply:
Will do. :D